workspace/Dockerfile-70

```diff
@@ -394,6 +394,85 @@ RUN if [ ${INSTALL_LINUXBREW} = true ]; then \
 ;fi
 
 #####################################
+# SQL SERVER:
+#####################################
+ARG INSTALL_MSSQL=true
```

Comment:
The `INSTALL_MSSQL` looks great here. It should be added on the following places to integrate it into Laradock:

- `docker-compose.yml` in the `workspace` > `build` > `args` section.
- `env-sample` in the `Containers Customization` > `WORKSPACE` section.